docs/global.css

```
@@ -262,31 +258,30 @@
 
 	margin-top: calc(var(--eds_spacing_large));
 	text-align: center;
+}
 
-	#radixFeaturesLinks ul {
-		display: flex;
-		justify-content: center;
-		align-items: baseline;
-		gap: 2rem;
+#radixFeaturesLinks {
+	display: flex;
```

```suggestion
	display: flex;
    flex-wrap: wrap;
```
Allow buttons in flex box to wrap over multiple lines if needed